src/auxiliary/auxiliary.jl

```diff
@@ -146,6 +146,22 @@ given in the Trixi documentation regarding unstructured meshes.
 default_example_unstructured() = joinpath(examples_dir(), "unstructured_2d_dgsem", "elixir_euler_basic.jl")
 
 
+"""
+    ode_default_options()
+
+Return the default options for OrdinaryDiffEq's `solve`. Pass `ode_default_options()...` to `solve`
+to only return the solution at the final time and enable **MPI aware** error-based step size control,
+whenever MPI is used.
+For example, use `solve(ode, alg; ode_default_options()...)`
+"""
+function ode_default_options()
+  if mpi_isparallel()
+    return (; save_everystep = false, internalnorm = ode_norm, unstable_check = ode_unstable_check)
+  else
+    return (; save_everystep = false)
+  end
```

Comment:
Note to self (no need to change this PR right now): This makes it type-unstable but that's already the case for all setup routines and MPI...